src/addons/editors/CheckboxEditor.js

```diff
@@ -12,6 +13,14 @@ const CheckboxEditor = React.createClass({
     dependentValues: React.PropTypes.object
   },
 
+  getValue(){
+    return this.getInputNode().value;
```

Comment:
We shouldn't get the value from the DOM, that's not a reactive approach, the checkbox value should be added to state and changed accordingly, in the `getValue` we just need to return the value saved in state.